setup.py

```diff
@@ -2447,8 +2447,13 @@ def split_var(name, sep):
         # features like DSO engines or external OSSL providers don't work.
         # Only tested on GCC and clang on X86_64.
         if os.environ.get("PY_UNSUPPORTED_OPENSSL_BUILD") == "static":
+            import subprocess, shutil
+            pkgconfig = os.getenv("PKG_CONFIG", shutil.which("pkg-config"))
```

Comment:
This will give incorrect results on systems without pkg-config. ``ax_check_openssl.m4`` works around missing pkg-config.

Reply:
That's fine, we don't support all platforms for this hack. We can fall back to the previous behaviour if there is no pkg-config